src/utils.ts

```diff
@@ -614,6 +614,58 @@ async function verifyFlavorPath(flavor: BrowserFlavor | undefined, platform: Pla
     }
 }
 
+type ExtensionSettings = [string, boolean | string | {[key: string]: string} | undefined];
+
+export function reportExtensionSettings(telemetryReporter: Readonly<TelemetryReporter>): void {
+    const extensionSettingsList = Object.entries(vscode.workspace.getConfiguration(SETTINGS_STORE_NAME)).splice(4) as Array<ExtensionSettings>;
+    const settings = vscode.workspace.getConfiguration(SETTINGS_STORE_NAME);
+    const changedSettingsMap: Map<string, string> = new Map<string, string>();
+    for (const currentSetting of extensionSettingsList) {
+        const settingName: string = currentSetting[0];
+        const settingValue: boolean | string | {[key: string]: string} | undefined = currentSetting[1];
+        const settingInspect = settings.inspect(settingName);
```

Comment:
`settingInspect` is used to grab the default value for a setting.  We use this to compare to the current value to catch any differences.